Packs/Slack/Integrations/Slack_IAM/Slack_IAM.py

```diff
@@ -483,7 +488,8 @@ def main():
     create_if_not_exists = demisto.params().get("create_if_not_exists")
 
     iam_command = IAMCommand(is_create_enabled, is_enable_enabled, is_disable_enabled, is_update_enabled,
-                             create_if_not_exists, mapper_in, mapper_out)
+                             create_if_not_exists, mapper_in, mapper_out,
+                             get_user_iam_attrs=['userName', 'emails'])
```

Comment:
why not search by id?

Reply:
@yaakovpraisler

Reply:
try using the following endpoint specifically for id: `scim/v2/Users/{id}`